VideoLocker/build.gradle

```diff
@@ -96,6 +96,7 @@ dependencies {
     compile 'com.android.support:cardview-v7:21.0.0'
     compile 'com.android.support:appcompat-v7:22.1.0'
     compile 'com.android.support:multidex:1.0.0'
+    compile 'com.github.bumptech.glide:glide:3.6.1'
```

Comment:
so, i guess i should just squash all the commits into one and give a detailed title+explanation to it.